lib/index.d.ts

```diff
@@ -0,0 +1,36 @@
+/* Type definitions for mobx-event-bus */
```

Comment:
This file shouldn't be placed under `lib`, but rather the root directory of the package. If you want it to live in some other directory, that should be explicitly specified in `package.json` using the "types" property (more info [here](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html)).